### ISSUE TITLE
Manually installation doesn't work

### ISSUE THREAD

**Issue by Gorael** [nikolaikopernik/code-complexity-plugin#26]
I tried to install the plugin manually and got the error "Fail to load plugin descriptor from file code-complexity-plugin-1.6.1.zip". The same error with other plugin versions. 
![image](https://github.com/user-attachments/assets/1f751abb-6a96-4c51-a313-ecc57f1134fd)

IntelliJ IDEA 2024.2.0.2
Build #IU-242.20224.419

**Comment by nikolaikopernik** [comment]
Indeed, the link leads to the archive with the sources, but in order to install it you need a JAR file. Will check it
